packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx

```diff
@@ -106,6 +106,69 @@ describe('VitalsHeader: ', () => {
     expect(mockLaunchWorkspace).toHaveBeenCalledTimes(1);
     expect(mockLaunchWorkspace).toHaveBeenCalledWith(patientVitalsBiometricsFormWorkspace);
   });
+
+  it('renders a normal header when all the values fall within normal range', async () => {
+    mockFhirVitalsResponse.entry[3].resource.valueQuantity.value = 79;
+    mockFhirVitalsResponse.entry[4].resource.valueQuantity.value = 119;
+    mockFhirVitalsResponse.entry[0].resource.valueQuantity.value = 69;
+    mockFhirVitalsResponse.entry[11].resource.valueQuantity.value = 36;
+    mockOpenmrsFetch.mockReturnValue({ data: mockFhirVitalsResponse });
+
+    renderVitalsHeader();
+
+    await waitForLoadingToFinish();
+
+    const expandButton = screen.getByTitle(/ChevronDown/);
+    userEvent.click(expandButton);
+
+    expect(screen.queryByTitle(/abnormal value/i)).not.toBeInTheDocument();
+  });
+
+  it('renders a header with a pink backgroung when the diastolic value falls below the configured low normal range', async () => {
```

Comment:
```suggestion
  it('flags abnormal values that lie outside of the provided reference ranges', async () => {
```

Reply:
This spec and the ones that follow it all seem to make the same assertion. It's probably better to combine them into one spec and make the different assertions within it.

Reply:
The other assertion could read something like:

```javascript
it(does not flag normal values that lie within the provided reference ranges, async () => {
```